README.md

```diff
@@ -376,7 +376,7 @@ conda install -c conda-forge nncf
 - Python\* 3.8 or later
 - Supported frameworks:
   - PyTorch\* >=2.2, <2.4
-  - TensorFlow\* >=2.8.4, <=2.12.1
+  - TensorFlow\* >=2.8.4, <=2.15.1
```

Comment:
Please update version in https://github.com/openvinotoolkit/nncf/blob/develop/docs/Installation.md?plain=1#L46

Reply:
@alexsu52 As far as I understand from our offline discussion, it was suggested not to modify Installation.md at that point. Am I right?

Reply:
I agree with @AlexanderDokuchaev's statement. The `installation.md should be updated. Could you suggest options?

Reply:
As I said before, my suggestion is as follows:

| NNCF      | OpenVINO   | PyTorch  | ONNX     | TensorFlow | Python |
|-----------|------------|----------|----------|------------|--------|
| `develop` | `2024.2.0` | `2.3.0`  | `1.16.0` | `2.15.1`   | `3.8`*  |

> (*) Python 3.9 or higher is required for TensorFlow 2.15.1

Reply:
Done